### PR TITLE
[WIP] Cross loss entropy cpp example

### PR DESCRIPTION
**Pull request by jjsjann123**

### CODE REVIEW

File: test/test_gpu3.cpp [hunk 1]
```diff
@@ -8099,6 +8099,69 @@ TEST_F(NVFuserTest, FusionManagedData_CUDA) {
...
+    auto tv6 = exp(tv5);
+    auto tv7 = sum(tv6, {1});
+    auto tv8 = broadcast(tv7, {false, true});
+    auto tv9 = expand(tv8, 
+                      {IrBuilder::create<Int>(128),
+                       IrBuilder::create<Int>(371)});
+    auto tv10 = div(tv6, tv9);
+    auto tv11 = log(tv10);
+    auto tv12 = neg(tv11);
+    auto tv13 = reshape(tv1, {128}, {128, 1});
```
Comment (naoyam) [inline test/test_gpu3.cpp:8127]: nit: we can use `unsqueeze`

Comment (jjsjann123) [reply test/test_gpu3.cpp:8127]: yeah, the only reason we are seeing a reshape here is that unsqueeze/squeeze is not in thunder lowering yet.

**Comment by jjsjann123** [comment]
tagging #156

File: test/test_gpu3.cpp [hunk 2]
```diff
@@ -8099,6 +8099,69 @@ TEST_F(NVFuserTest, FusionManagedData_CUDA) {
...
+  {
+    auto t0 = at::randn({128, 371}, options);
+    inputs.push_back(t0);
+    auto t1 = at::randint(371, {128}, options).to(at::ScalarType::Long);
+    inputs.push_back(t1);
+    // note: reduction arg
+    //   none -> 0
+    //   mean -> 1
+    //   sum  -> 2
+    auto t24 = at::cross_entropy_loss_symint(t0, t1, None, 1, 5, 0.0);
```
Comment (naoyam) [inline test/test_gpu3.cpp:8155]: `None` is not valid in C++. What's the right way to call this function?

Comment (naoyam) [reply test/test_gpu3.cpp:8155]: Never mind, it's just `{}`

Comment (jjsjann123) [reply test/test_gpu3.cpp:8155]: Oops. sorry about this, must have copied the wrong file :bow:

**Comment by jjsjann123** [comment]
closing this one since Naoya is already on top of it.

**Pull request closed without merging**